<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="port_info_label">
     <property name="text">
      <string>PORT_INFO</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="port_info_count"/>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="dask_node_connect_seconds"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="check_worker_started_label">
     <property name="text">
      <string>CHECK_WORKER_STARTED</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="jupyter_json_seconds"/>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="scheduler_connect_seconds"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="jupyter_json_label">
     <property name="text">
      <string>JUPYTER_JSON</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="deploy_dask_scheduler_label">
     <property name="text">
      <string>DEPLOY_DASK_SCHEDULER</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="deploy_dask_scheduler_seconds"/>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="jupyter_json_count"/>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="deploy_dask_worker_seconds"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="get_scheduler_address_label">
     <property name="text">
      <string>GET_SCHEDULER_ADDRESS</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="get_scheduler_address_count"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="deploy_dask_worker_label">
     <property name="text">
      <string>DEPLOY_DASK_WORKER</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="dask_node_connect_label">
     <property name="text">
      <string>DASK_NODE_CONNECT</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="deploy_dask_scheduler_count"/>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="dask_node_connect_count"/>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="port_info_seconds"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="scheduler_connect_label">
     <property name="text">
      <string>SCHEDULER_CONNECT</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="deploy_dask_worker_count"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="scheduler_connect_count"/>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="get_scheduler_address_seconds"/>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="check_worker_started_count"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="cluster_names_box"/>
   </item>
   <item row="14" column="1">
    <widget class="QSpinBox" name="tunnel_try_again_with_any_port_count"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="validate_http_tunnel_label">
     <property name="text">
      <string>VALIDATE_HTTP_TUNNEL</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="squeue_after_sbatch_count"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="squeue_after_sbatch_label">
     <property name="text">
      <string>SQUEUE_AFTER_SBATCH</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QSpinBox" name="tunnel_try_again_with_any_port_seconds"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="open_tunnel_label">
     <property name="text">
      <string>OPEN_TUNNEL</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="cancel_deployment_seconds"/>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="check_worker_started_seconds"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="cancel_deployment_label">
     <property name="text">
      <string>CANCEL_DEPLOYMENT</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="open_tunnel_count"/>
   </item>
   <item row="13" column="1">
    <widget class="QSpinBox" name="validate_http_tunnel_count"/>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="cancel_deployment_count"/>
   </item>
   <item row="12" column="2">
    <widget class="QSpinBox" name="open_tunnel_seconds"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="tunnel_try_again_with_any_port_label">
     <property name="text">
      <string>TUNNEL_TRY_AGAIN_WITH_ANY_PORT</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSpinBox" name="squeue_after_sbatch_seconds"/>
   </item>
   <item row="13" column="2">
    <widget class="QSpinBox" name="validate_http_tunnel_seconds"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="cluster_name_addc_label_29">
     <property name="text">
      <string>Count</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="cluster_name_addc_label_30">
     <property name="text">
      <string>Seconds between</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QPushButton" name="save_timeouts_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save timeouts</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
